staging/src/k8s.io/apimachinery/Godeps/Godeps.json

```diff
@@ -213,6 +213,10 @@
 		{
 			"ImportPath": "gopkg.in/yaml.v2",
 			"Rev": "53feefa2559fb8dfa8d81baad31be332c97d6c77"
+		},
+		{
+			"ImportPath": "k8s.io/kube-openapi/pkg/common",
```

Comment:
Didn't see this coming.  @sttts How did apimachinery come to depend on kube-openapi?

Reply:
One of the imports is for this:

```
func (_ Time) OpenAPIDefinition() openapi.OpenAPIDefinition {
        return openapi.OpenAPIDefinition{
                Schema: spec.Schema{
                        SchemaProps: spec.SchemaProps{
                                Type:   []string{"string"},
                                Format: "date-time",
                        },
                },
        }
}
```